src/AzureEventGridSimulator/Program.cs

```diff
@@ -14,46 +14,53 @@ namespace AzureEventGridSimulator
 {
     public static class Program
     {
-        public static void Main()
+        public static void Main(string[] args)
         {
             try
             {
-                var host = WebHost.CreateDefaultBuilder()
-                                  .ConfigureAppConfiguration((context, builder) =>
-                                  {
-                                      var env = context.HostingEnvironment;
+                var host = WebHost
+                    .CreateDefaultBuilder(args)
+                    .ConfigureAppConfiguration((context, builder) =>
+                    {
+                        var env = context.HostingEnvironment;
 
-                                      var config = builder.AddJsonFile("appsettings.json", false, false)
-                                                          .AddJsonFile($"appsettings.{env.EnvironmentName.ToLowerInvariant().Trim()}.json", true, false)
-                                                          .AddEnvironmentVariables()
-                                                          .Build();
+                        var config = builder.Build();
```

Comment:
By calling `CreateDefaultBuilder`, the configuration already reads from these 3 sources. No need to add them again. The only difference now is that `appsettings.json` is no longer required (by default it is added as optional)